source/js/buyers-guide/search/search-filter.js

```diff
@@ -42,9 +42,9 @@ export class SearchFilter {
         )
     ).then(() => {
       if (this.categoryTitle.value === "None") {
-        Utils.toggleScrollAnimation();
+        Utils.toggleScrollAnimation(true);
       } else {
-        Utils.toggleCategoryAnimation();
+        Utils.toggleCategoryAnimation(true);
       }
```

Comment:
This isn't true, though. Initial load is not base don "is there a category", it's `true` by default, and then needs to get set to `false`. So in the constructor, `this.initialLoad = true`, and then in this `then` handler:
```js
    .then(() => {
      if (this.categoryTitle.value === "None") {
        Utils.toggleScrollAnimation(this.initialLoad);
      } else {
        Utils.toggleCategoryAnimation(this.initialLoad);
      }
      this.initialLoad = false;
    });
```